cosmovisor/args.go

```diff
@@ -184,7 +183,7 @@ func GetConfigFromEnv() (*Config, error) {
 }
 
 // LogConfigOrError logs either the config details or the error.
-func LogConfigOrError(logger zerolog.Logger, cfg *Config, err error) {
+func LogConfigOrError(logger *logging.Logger, cfg *Config, err error) {
```

Comment:
Why are we using this type over zerolog directly?

Reply:
Maybe so is more pluggable? but I also wonder the same

Reply:
Well, the goal was to be able to change the `Logger` implementation without needing to refactor the codebase.
I realize now that we actually still need to refactor the log call..

Reply:
I'm not sure this is the correct approach. We should either use zerolog directly or if anything, just do what the SDK does and use the `tmlog.Logger` interface instead of creating a wrapper.

Reply:
Yeah, probably an alternative approach is using an interface and making it interchangeable by using a factory.